ceph-releases/kraken/centos/7/demo/README.md

```diff
@@ -0,0 +1 @@
+../../../ubuntu/14.04/demo/README.md
```

Comment:
The link points to a non-existing file. Just copy the jewel file for the demo container.